Tangem/Modules/Send/StakingNotifications/StakingNotificationManager.swift

```diff
@@ -86,6 +86,10 @@ private extension CommonStakingNotificationManager {
                 )
             }
 
+            if !tokenItem.supportsStakingOnDifferentValidators, readyToStake.stakeOnDifferentValidator {
```

Comment:
я бы лучше у yield проверил прям по месту

```suggestion
            if yield.item.network == .tron, readyToStake.stakeOnDifferentValidator {
```

Reply:
тут подстелил соломки если еще у какой-то сети будет подобное поведение